src/components/LinksSection.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from "prop-types";
+
+const LinksSection = ({ className, linksList }) => {
+  return (
+    <div className={className}>
+      <h3 className="pb-10 text-2xl font-bold uppercase font-heading">Links</h3>
+      <div className="flex flex-col gap-5 justify-center items-center w-full lg:flex-row lg:gap-10">
+        {linksList.map(({ title, target }) => (
+          <a
+            className={`flex items-center justify-center py-2 px-5 rounded-lg font-heading font-normal text-center border-4 border-orange-600 shadow-md hover:bg-orange-600 hover:border-orange-600 w-full text-nowrap`}
```

Comment:
this is still not fixed, a templated string is not needed here since you're not rendering anything inside, using double quotes here is fine

Reply:
a couple notes about the classes
* why are `items-center` and `justify-center` here, don't think there's anything to align here is there?
* don't think we want `text-nowrap` here since we don't want text to just overflow off the button
* don't think `font-normal` is needed since that's the default
* nit: I would use `orange-500` instead of `orange-600` as default (also to be consistent with the rest of the Arise components), see https://www.refactoringui.com/previews/building-your-color-palette for why/how Tailwind thinks about colors in the design system

Reply:
i think you might be intending for the effect of `flex-wrap` instead of `text-nowrap`? https://tailwindcss.com/docs/flex-wrap

Reply:
also I'm thinking it might be good to not have the buttons be full width with `w-full` but instead set a min width with `min-w-xs` or something when they are vertically stacked? that might be a question for the design team

Reply:
i used text-nowrap because Praise Report is pretty long and it will cause the button text to show 2 lines. it looks out of place among the rest

Reply:
the `min-w-fit` you added fixes that so `text-nowrap` should be good to remove, generally `text-nowrap` is probably not what you're looking for since you're telling the contents to spill out of the container which is most likely not what you want. also, don't think `items-center` and `justify-center` are being used here so we should remove that. might be good to add `mx-auto` here to center the buttons